bcgov/cas-registration
# feature: storage of petroleum products activity page

Card: https://github.com/bcgov/cas-reporting/issues/338
Changes:-

- Adds backend config/migration for Storage of petroleum products activity.
- Adds frontend page and components for Storage of petroleum products activity.


To test:-
- make reset_db to apply latest migrations.
- spin up the front, backend and postgres.
- Login, then navigate to http://localhost:3000/reporting/reports , select operation 3 -> press save and continue to go to review Facilities page-> select Storage of petroleum products(from the list of activities) and press save & continue. Make sure Storage of petroleum products is checked in the facility review page before progressing to the activities.
- The first 2 methodology options have no dependant fields, and the last 2 both have description as a dependant field
